recipes/nest-desktop/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "nest-desktop" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nest-desktop-{{ version }}.tar.gz
+  sha256: 49051aeefc8be189ae7dfb64ad983b7a0a5636765a79febd3347194b57ba61ba
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  entry_points:
+    - nest_desktop = nest_desktop.app:run
```

Comment:
```suggestion
```
Actually, your app also doesn't have this entry point because it is not registered in your setup.py.

Reply:
Ok, we do not have any entry points. Thus, we will remove it.

Reply:
Yes, please remove these lines because your app currently does not have a console_script entry_point.